src/main/java/com/salesforce/dataloader/security/EncryptionUtil.java

```diff
@@ -149,17 +164,17 @@ private static Cipher createCipher() throws NoSuchAlgorithmException, NoSuchPadd
     /**
      * Encrypt a string using whatever is the default encryption technique for the system.
      *
-     * @param clearText
      * @return The encrypted string
-     * @throws GeneralSecurityException
      */
     public String encryptString(String clearText) throws GeneralSecurityException {
         if (clearText == null) return clearText;
         Key oKey = getCryptoKey();
         Cipher oCipher = createCipher();
         byte[] encryptedBytes;
 
-        if (oCipher == null) { return clearText; }
+        if (oCipher == null) {
+            return clearText;
```

Comment:
I didn't notice this before, and this whole function needs to go away anyway, but just for the record: don't do this! The correct thing to do if you can't encrypt the string is to throw an exception. If for some reason you *must* return a value, return `null`. Returning the plaintext string as the output of an encryption function under *any* circumstance creates a risk of exposing the secret data that is supposed to be protected.

Reply:
Agreed, let's remove this. Leaving in dead code will only make future maintenance harder.